### PR TITLE
Ensure we always add the last applied annotation

### PR DESCRIPTION
**Pull request by JoelSpeed**
The last applied annotation should be applied even when creating instances. I believe this may have caused a bug faced by @mthssdrbrg earlier this week where the three way merge repeatedly added a line where Quack templating was differing

### CODE REVIEW

File: pkg/controller/gittrack/gittrack_controller_test.go [hunk 1]
```diff
@@ -671,12 +685,12 @@ var _ = Describe("GitTrack Suite", func() {
...
 				var hist prometheus.Histogram
 				Eventually(func() error {
-					var err error
-					hist, err = metrics.TimeToDeploy.GetMetricWith(map[string]string{
+					observer, err := metrics.TimeToDeploy.GetMetricWith(map[string]string{
 						"name":       instance.GetName(),
 						"namespace":  instance.GetNamespace(),
 						"repository": instance.Spec.Repository,
 					})
+					hist = observer.(prometheus.Histogram)
```
Comment (mthssdrbrg) [inline pkg/controller/gittrack/gittrack_controller_test.go]: What's up with these changes? They don't seem related?

Comment (JoelSpeed) [reply pkg/controller/gittrack/gittrack_controller_test.go]: Result of a dirty `vendor` (from testing a newer version of `prometheus/client_golang`), have removed these changes and rebased
